### PR TITLE
Updates to Kea DHCPv6 PD route hook

### PR DESCRIPTION
**Pull request by cbuechler**
Add needed lease6_rebind. Fix route deletion errors when interface is missing. Add a bit of logging so actions are visible. Clarify variable names (`lease` is more appropriately descriptive than `client`, IMO).

### CODE REVIEW

File: src/system/on-dhcpv6-event.sh [hunk 1]
```diff
@@ -31,38 +31,47 @@ case "$action" in
...
+        route_cmd+=" dev ${ifname}"
+    fi
+    route_cmd+=" proto static"
+    eval "$route_cmd"
 
     exit 0
     ;;
 
-  leases6_committed)
+  leases6_committed|lease6_rebind)
```
Comment (sarthurdev) [inline src/system/on-dhcpv6-event.sh]: `lease6_rebind` not required here. The `leases6_committed` hook is called after each event and is the only hook where the array of leases (LEASE6_AT[num]_) is available.

**Review by sarthurdev — CHANGES_REQUESTED** [review]
Happy to merge the renamed variables and ifname check.

File: src/system/on-dhcpv6-event.sh (hunk 1 above)
Comment (cbuechler) [reply src/system/on-dhcpv6-event.sh:55]: I forgot to explain why that's there. It'll miss route additions without `lease6_rebind` in cases where clients are renewing and a route isn't already present. That'll happen in two real world scenarios (saw it myself, which is why I added). 

1) After upgrading to a version of VyOS including this new feature, routes won't be added until the downstream devices go back through SOLICIT from scratch. This generally won't happen unless forced, or potentially via a reboot depending on whether the client maintains its DHCPv6 lease info persistently. 
2) After removing `disable-route-autoinstall` config node, it ends up in the same scenario as 1.

Comment (cbuechler) [reply src/system/on-dhcpv6-event.sh]: You're right, I had a problem initially that I thought this solved, but was unrelated on closer review. Reverted that.

**Pull request merged**